crates/telio-wg/src/wg.rs

```diff
@@ -605,7 +605,7 @@ impl State {
 
             // Node is new and default LinkState is down. Save it before sending the event
             self.no_link_detection
-                .insert(key, None, None, PeerState::Connecting);
+                .insert(key, peer.rx_bytes, peer.tx_bytes, PeerState::Connecting);
```

Comment:
Why are we saving tx and rx bytes here. I did not understand this.

Reply:
This is a simplification - instead of assuming that the wireguard always starts with `None` for a fresh peer, we use the actual value as returned by the wireguard implementation (even if most of the time wireguard will indeed start from `None`).